test/cloud_testing/platforms/slc6_x86_64_test.sh

```diff
@@ -73,12 +73,68 @@ CVMFS_TEST_CLASS_NAME=ServerIntegrationTests                                  \
                               -x src/518-hardlinkstresstest                   \
                                  src/524-corruptmanifestfailover              \
                                  src/585-xattrs                               \
+                                 src/700-overlayfsvalidation                  \
                                  --                                           \
                                  src/5*                                       \
                                  src/6*                                       \
+                                 src/7*                                       \
                               || retval=1
 
 
+echo -n "starting FakeS3 service... "
```

Comment:
Alright. I will go through all the test scripts and disable the S3 tests, to reduce the noise in the test output.